docs/generated/.files_generated

```diff
@@ -0,0 +1,2 @@
+oadm_by_example_content.adoc
+oc_by_example_content.adoc
```

Comment:
Keeps a list of which files in a directory are generated (you may have others in the same dir). It's used by the generation script in `hack/common.sh`, it does the same for bash completions and `kubectl` docs upstream.